plugins/modules/digital_ocean_droplet_info.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Ansible Project
+# Copyright: (c) 2020, Tyler Auerbeck <tauerbec@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: digital_ocean_droplet_info
+short_description: Gather information about DigitalOcean Droplets
+description:
+    - This module can be used to gather information about Droplets.
+author: "Tyler Auerbeck (@tylerauerbeck)"
+version_added: 1.2.0
```

Comment:
Forgot this, next version will be [1.4.0](https://github.com/ansible-collections/community.digitalocean/releases) if you wouldn't mind changing it -- or, I can fix it up in the `release` branch if I remember.

Reply:
And, apologies for the back-and-forth, I've been multitasking too much lately.

Reply:
No problems! Fixed.